### PR TITLE
Story/14557 configurable status tab notice

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
- New tile for ramp-down notification added to homepage according to Figma
- The visibility of the new tile shall be configurable via server configuration (CCL)
- Clicking on the tile opens a detail screen with details about the ramp-down

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14572
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14573
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14619
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14641

## Screenshots
WhatsApp Image 2023-01-26 at 16 23 11
WhatsApp Image 2023-01-26 at 16 23 11 (1)
WhatsApp Image 2023-01-26 at 16 23 11 (3)
WhatsApp Image 2023-01-26 at 16 23 11 (2)

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			localizedText: ["de": "Der Betrieb der Corona-Warn-App wird am xx.xx.xxxx eingestellt."],
+			parameters: []
+		)
+		
+		let longText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risikobegegnungen und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift]: I know that's only the test but still I suggest to change "Risikobegegnungen" to "Risiko-Begegnungen".
```suggestion
			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risiko-Begegnungen und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One suggestion

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
+//
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift:1]: This test fails on my machine. Is it only me?

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/AppClosureNoticeDetails/__tests__/AppClosureNoticeDetailsViewModelTests.swift:1]: It is passing on the pipeline.

File: src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift [hunk 3]
```diff
@@ -309,6 +315,39 @@ class CCLService: CCLServable {
...
+			localizedText: ["de": "Der Betrieb der Corona-Warn-App wird am xx.xx.xxxx eingestellt."],
+			parameters: []
+		)
+		
+		let longText = DCCUIText(
+			type: "string",
+			quantity: nil,
+			quantityParameterIndex: nil,
+			functionName: nil,
+			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risikobegegnungen und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift]:
```suggestion
			localizedText: ["de": "Sie erhalten dann keine Warnungen mehr über Risiko-Begegnung und können selbst andere nicht mehr warnen. Sie können keine Tests mehr registrieren und erhalten keine Testergebnisse mehr über die App. Auf Ihre Zertifikate und das Kontakt-Tagebuch haben Sie weiterhin Zugriff. Allerdings können Sie keine neuen Zertifikate mehr hinzufügen."],
```

Comment (Ein-Tim) [reply src/xcode/ENA/ENA/Source/Services/CCLService/CCLService.swift]: @naveeddotio This should be an uppercase "B"

**Review by puneetmahali1 — APPROVED** [review]

**Pull request merged**